src/pybind/mgr/stats/fs/perf_stats.py

```diff
@@ -126,7 +126,12 @@ def set_client_metadata(self, client_id, key, meta):
     def notify(self, cmdtag):
         self.log.debug("cmdtag={0}".format(cmdtag))
         with self.meta_lock:
-            result = self.client_metadata['in_progress'].pop(cmdtag)
+            try:
+                result = self.client_metadata['in_progress'].pop(cmdtag)
+            except KeyError:
+                self.log.warn("cmdtag not found in client metadata")
```

Comment:
```suggestion
                self.log.warn(f"cmdtag {cmdtag}  not found in client metadata")
```

Reply:
We are **mostly** expecting empty values for `cmdtag` here. So this change doesn't make a difference.

Reply:
I've fixed it anyway. It works better if a uuid is not found, somehow. Thanks Seb.